src/common.ts

```diff
@@ -53,25 +54,41 @@ export class WrappedContract {
   }
 }
 
+export enum ContractAddressId {
+  oracle = "ORACLE",
+  lens = "LENS",
```

Comment:
Don't need this one actually, see https://github.com/yearn/yearn-sdk/issues/228

Reply:
Removed lens